spring-web/src/main/java/org/springframework/http/converter/ResourceRegionHttpMessageConverter.java

```diff
@@ -227,14 +219,14 @@ private void writeResourceRegionCollection(Collection<ResourceRegion> resourceRe
 					println(out);
 				}
 				long resourceLength = region.getResource().contentLength();
-				end = Math.min(end, resourceLength - inputStreamPosition - 1);
+				end = min(end, resourceLength - inputStreamPosition - 1);
 				print(out, "Content-Range: bytes " +
 						region.getPosition() + '-' + (region.getPosition() + region.getCount() - 1) +
 						'/' + resourceLength);
 				println(out);
 				println(out);
 				// Printing content
-				StreamUtils.copyRange(in, out, start, end);
+				copyRange(in, out, start, end);
```

Comment:
its random impl. detail where this comes from we want to copyRange who how or what seems different concern.